.github/workflows/ops-application-onboarding-or-promotion-form.yml

```diff
@@ -0,0 +1,60 @@
+name: Application onboarding or promotion
+
+on:
+  issues:
+    types:
+      - opened
```

Comment:
Is this workflow going to run on every issue opened within the `va.gov-team` repo?

Reply:
It will run, check to see if the title includes one of the two options after this and skip the rest if it doesn't match. So pretty negligible on timing, resources, and run-minutes.